### ISSUE TITLE
Consider making callbackFn always the last parameter

### ISSUE THREAD

**Issue by AndrewRayCode** [sheebz/phantom-proxy#15]
`evaluate(functionToEvaluate, callbackFn, [arg1, arg2,... argN]`

I think this would be better written as:

`evaluate(functionToEvaluate, [arg1, arg2,... argN], callbackFn`

It's more node-ish, and it's more phantom-ish

**Comment by sheebz** [comment]
Yeah, I think that makes sense.  This would be a breaking change though, I'm hesitant to implement.

**Comment by kevzettler** [comment]
This is standard node.js convention.

**Comment by sheebz** [comment]
@kevzettler, yeah I wasn't aware of this when I originally wrote fn.  If there is enough interest I will change it.

**Comment by sandinmyjoints** [comment]
+1
